distatis_weights_upgma.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Run this file using 'qsub distatis_weights_upgma.sh path/to/input.fasta path/to/output_alignment.txt path/to/output_matrix_test.txt path/to/output_matrix.csv'
+
+# All lines starting with "#$" are SGE qsub commands
+
+# Specify a project to run under
+#$ -P bi720
+
+# Give this job a name
+#$ -N distatis_weights_upgma
+
+# Join standard output and error to a single file
+#$ -j y
+
+# Name the file where to redirect standard output and error
+#$ -o /projectnb/bi720/MMG/2fungi2furious/hpenn13/compbio-project/logs/distatis_weights_upgma.log
```

Comment:
Make log file a relative path (@tamarahadz added something to another branch that will create the `logs` folder if it doesn't exist)